### PR TITLE
UI Improvement: Transformer Inputs

### PR DESCRIPTION
**Pull request by ariefrahmansyah**
**What this PR does / why we need it**:

1. On specifying table inputs, users can specify columns without the need to specify the base table.

**Before**

image

**After**

image

2. On specifying feast inputs, users can group feast tables so the retrieval process started asynchronously.

**Before**

image

image

**After**

image

image

**Checklist**

- [x] Tested locally

### CODE REVIEW

**Comment by pradithya** [comment]
So if there are 2 or more feast input that's defined subsequent to each other, these will be flattened into 1 `feast` input?
But will not be flattened if it is separated by other input?

**Comment by pradithya** [comment]
> So if there are 2 or more feast input that's defined subsequent to each other, these will be flattened into 1 `feast` input?
> But will not be flattened if it is separated by other input?

I think the screenshot in description is misleading

**Comment by ariefrahmansyah** [comment]
> So if there are 2 or more feast input that's defined subsequent to each other, these will be flattened into 1 `feast` input?
> But will not be flattened if it is separated by other input?

No. It's not automatically flattened. The user must manually click `+ Add Feast Table` to have another Feat retrieval on the same group.

image

**Comment by pradithya** [comment]
> > So if there are 2 or more feast input that's defined subsequent to each other, these will be flattened into 1 `feast` input?
> > But will not be flattened if it is separated by other input?
> 
> No. It's not automatically flattened. The user must manually click `+ Add Feast Table` to have another Feat retrieval on the same group.
> 
> image

Yes, the button is not visible in previous screenshot. It looks good to me

**Review by pradithya — APPROVED** [review]
LGTM!

**Comment by ariefrahmansyah** [comment]
I've added more screenshots on the description.

File: ui/src/pages/version/components/forms/components/feast_config/components/FeastInputGroup.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+          </EuiDragDropContext>
+        </EuiFlexGroup>
+
+        <EuiFlexItem>
+          <AddButton
+            title="+ Add Feast Table"
+            description="Add another Feast table that will be created asynchronously with other tables in this group."
+            titleSize="xs"
+            onClick={() => onAddTable()}
+          />
```
Comment (tiopramayudi) [inline ui/src/pages/version/components/forms/components/feast_config/components/FeastInputGroup.js:100]: Please remove this if not being used

**Comment by tiopramayudi** [comment]
left one comment, the rest is LGTM

**Pull request merged**